setup.py

```diff
@@ -11,8 +11,9 @@
 SOURCE_DIR = os.path.join(ROOT_DIR)
 
 requirements = [
-    'websocket-client >= 0.32.0',
-    'requests >= 2.14.2, != 2.18.0',
+    'websocket-client >= 0.32.0, < 1',
+    'requests >= 2.14.2, != 2.18.0, < 3',
+    'six >= 1.16.0, < 2',
```

Comment:
This should not be added here, this was resolved in https://github.com/docker/docker-py/commit/a9748a8b702a3c75b46ba8c8d0490e4b8ec5ab04

Reply:
@JonZeolla what's the best way to get people's attention to PRs? 😅